### ISSUE TITLE
CTRL+SHIFT+P -> Docs: Snippet command inserts invalid markup when using Id

### ISSUE THREAD

**Issue by jasonjoh** [microsoft/vscode-docs-authoring#470]
docs-markdown version 0.2.59, using Visual Studio Code Insiders 1.44.0-insiders

Steps to reproduce:

1. CTRL+SHIFT+P, choose **Docs: Snippet**
1. Choose **Full Search**
1. Enter a valid file name to a code file in the repository
1. Choose the file
1. Choose **Id**
1. Enter a value for the snippet ID

The resulting markup that's inserted is missing a double-quote in front of the snippet ID, which causes the build to fail.

Example (note that `MySnippet` should have a leading double-quote):

```
:::code language="csharp" source="../demo/GraphTutorial/Program.cs" id=MySnippet":::
```

**Comment by bharney0** [comment]
@jasonjoh Thank you for reporting this problem. We have reproduced this bug and will fix the problem as soon as possible. We will reach out to you if we have more questions.

**Comment by bharney0** [comment]
Bug fix done on PR #474
